packaging/fpm/Dockerfile

```
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: AGPL-3.0-only
+# Provenance-includes-location: https://github.com/grafana/mimir/packaging/fpm/Dockerfile
+# Provenance-includes-license: Apache-2.0
+# Provenance-includes-copyright: Grafana Labs.
+
+FROM alpine:3.8
```

To be consistent with our other Dockerfiles.

```suggestion
FROM alpine:3.16.0
```